README.md

```diff
@@ -50,7 +50,7 @@ languages in [SonarQube](http://www.sonarqube.org/), [SonarCloud](https://sonarc
 ## How to contribute
 
 There are many ways you can contribute to the `sonar-dotnet` project. 
-When contributing, please respect our [Code of Conduct](./CODE_OF_CONDUCT.md).
+When contributing, please respect our [Code of Conduct](./docs/code-of-conduct.md).

```

Comment:
This is just cleanup of the root directory (moved file).

And the end of this file, there's `## License` section. That's probably fine (?).

Reply:
I'd keep the license in the root, as it is

Reply:
Does it imply that you want to remove the `## License` section from `readme.md`? As `license.txt` file is discussed below.

Reply:
I don't mind either way, TBH, we can leave it as is